jsamchineme/sendit-react
# #165061191 - Add badges

#### What does this PR do?
- Add badges to readme
#### Description of Task to be completed?
- Add travis build badage
- Add codeclimate maintainability badge
- Add coverall badge
#### How should this be manually tested?
- pull branch
- check readme
#### Any background context you want to provide?
- None
#### What are the relevant pivotal tracker stories?
- https://www.pivotaltracker.com/story/show/165061191
#### Screenshots (if appropriate)
None
#### Questions:`
None
